app/src/main/java/com/cornellappdev/uplift/ui/components/gymdetail/GymSwimmingSection.kt

```diff
@@ -0,0 +1,97 @@
+package com.cornellappdev.uplift.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.models.Gym
+import com.cornellappdev.uplift.util.ACCENT_CLOSED
+import com.cornellappdev.uplift.util.LIGHT_YELLOW
+import com.cornellappdev.uplift.util.PRIMARY_YELLOW
+import com.cornellappdev.uplift.util.montserratFamily
+
+/**
+ * Displays swimming information for the given gym throughout the week, defaulting to the day
+ * given by parameter [today].
+ */
+@Composable
+fun GymSwimmingSection(today: Int, gym: Gym) {
```

Comment:
general question, is there normally multiple times in a day for a gym.

Reply:
Not generally, no, but I designed everything with as much flexibility as possible to be able to handle (hopefully) all cases. I guess it gets a bit messy in some places, but I hope my documentation accounts for that.